src/components/common/FeatureBox/FeatureBox.module.scss

```diff
@@ -4,6 +4,8 @@
   border: 1px solid $root-border-color;
   text-align: center;
   margin-top: 40px;
+  height: 70%;
```

Comment:
A dlaczego 70% akurat? Jak dodam dużo tekstu w środku to się rozjeżdża, proponuję dać 100% i marginesem zrobić odstęp ;)